lib/roo/excelx/sheet_doc.rb

```diff
@@ -1,11 +1,12 @@
+# frozen_string_literal: true
 require 'forwardable'
 require 'roo/excelx/extractor'
 
 module Roo
   class Excelx
     class SheetDoc < Excelx::Extractor
       extend Forwardable
-      delegate [:styles, :workbook, :shared_strings, :base_date] => :@shared
```

Comment:
Again avoided using delegated method(which are called almost once per cell) because it will result in creating a blank array for *args. I'm open to creating helper method like delegate_without_argument if that is more acceptable.